controllers/articles.controllers.js

```diff
@@ -0,0 +1,34 @@
+const { getThisArticle, incrementArticleVotes } = require("../models/articles.models")
+
+exports.getArticle = (req, res, next) => {
+    getThisArticle(req.params.article_id)
+    .then(article => {
+        res.status(200).send({article})
+    })
+    .catch(err => {
+        next(err, req.params.article_id)
+    })
+
+}
+
+exports.updateArticleVotes = (req, res, next) => {
+    if(Object.keys(req.body).length === 0 || !req.body.hasOwnProperty('inc_votes')) {
+        return Promise.reject({status: 400, msg: "inc_votes must be provided"})
+        .catch(err => {
```

Comment:
All this logic, needs to be exported to your model, the controllers job is to handle the req and res.
This way we don't need to chain more catch blocks on